<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramaFormDialog</class>
 <widget class="QDialog" name="ProgramaFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario de Programa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfoBasica">
      <attribute name="title">
       <string> Informaci贸n B谩sica</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Datos del Programa</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelCodigo">
            <property name="text">
             <string>C贸digo*:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtCodigo">
            <property name="placeholderText">
             <string>Ej: DIP-001, CUR-2024</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelNombre">
            <property name="text">
             <string>Nombre*:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtNombre">
            <property name="placeholderText">
             <string>Nombre completo del programa</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelDescripcion">
            <property name="text">
             <string>Descripci贸n:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTextEdit" name="txtDescripcion"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelDuracion">
            <property name="text">
             <string>Duraci贸n (meses):</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinDuracionMeses">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>36</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelHoras">
            <property name="text">
             <string>Horas totales:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinHorasTotales">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelCupos">
            <property name="text">
             <string>Cupos totales*:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spinCuposTotales">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCostos">
      <attribute name="title">
       <string> Costos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Estructura de Costos</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelCostoInscripcion">
            <property name="text">
             <string>Inscripci贸n (Bs.):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinCostoInscripcion">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="prefix">
             <string>Bs. </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCostoMatricula">
            <property name="text">
             <string>Matr铆cula (Bs.):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinCostoMatricula">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="prefix">
             <string>Bs. </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelCostoColegiatura">
            <property name="text">
             <string>Colegiatura (Bs.)*:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinCostoBase">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>50000</number>
            </property>
            <property name="value">
             <number>2500</number>
            </property>
            <property name="prefix">
             <string>Bs. </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelDescuentoContado">
            <property name="text">
             <string>Descuento contado (%):</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spinDescuentoContado">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="suffix">
             <string> %</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelResumenCostos">
         <property name="text">
          <string>Resumen de costos aparecer谩 aqu铆...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #f8f9fa;
    padding: 15px;
    border-radius: 5px;
    border: 1px solid #ddd;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConfiguracion">
      <attribute name="title">
       <string>锔 Configuraci贸n</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Configuraci贸n Adicional</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelEstado">
            <property name="text">
             <string>Estado:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboEstado">
            <item>
             <property name="text">
              <string>PLANIFICADO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>INICIADO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CONCLUIDO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CANCELADO</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFechas">
            <property name="text">
             <string>Fechas:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelFechaInicio">
              <property name="text">
               <string>Inicio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateFechaInicio"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelFechaFin">
              <property name="text">
               <string>Fin:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateFechaFin"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelCuotas">
            <property name="text">
             <string>Cuotas:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkCuotasMensuales">
              <property name="text">
               <string>Mensuales</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelDiasEntreCuotas">
              <property name="text">
               <string>D铆as entre cuotas:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinDiasEntreCuotas">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="value">
               <number>30</number>
              </property>
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkCuotasMensuales</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinDiasEntreCuotas</receiver>
   <slot>setDisabled(bool)</slot>
  </connection>
 </connections>
</ui>